app/assets/stylesheets/pages/_roasts.scss

```diff
@@ -0,0 +1,256 @@
+    margin-bottom: 10%;
+    width: 100%;
+    align-items: center;
+    .slider-axis {
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+      width: 61%;
+      margin-bottom: 3%;
+      @include breakpoint(300px 1000px) {
+        justify-content: space-around;
+        margin-bottom: 4%;
+      }
+      @include breakpoint($large-screens) {
+        width: 100%;
+      }
+      span {
+        display: flex;
+        flex-direction: row;
+        justify-content: space-evenly;
+        &.desktop-only {
+          display: none;
+          @include breakpoint($large-screens) {
+            display: flex;
+          }
+        }
+        @include fluid-size('width', 50px, 75px);
+        height: 15px;
+        span {
+          width: 50%;
+          border-right: solid 2.3px $range-track-color;
+        }
+        &.axial-last {
+          border: none;
+        }
+        @include breakpoint($large-screens) {
+          @include fluid-sizeFULL('width', 10px, 120px);
+          justify-content: flex-start;
+        }
+      }
+      img {
+        display: flex;
+        @include fluid-size('max-width', 21px, 31px);
+        // max-width: 31px;
+        margin-left: auto;
+        margin-right: auto;
+
+        @include breakpoint($large-screens) {
+          display: none;
+        }
+      }
+      .axial-text {
+        border: none;
+        color: $range-track-color;
+        display: none;
+        width: initial;
+        height: initial;
+        @include breakpoint($large-screens) {
+          @include fluid-size('font-size', 14px, 28px);
+          display: flex;
+          position: relative;
+        }
+      }
+    }
+    .slider {
+      display: flex;
+      flex-direction: row;
+      width: 64%;
+      @include breakpoint($large-screens) {
+        width: 90%;
+      }
+      img {
+        display: none;
+        align-self: center;
+        margin-left: 1.2vw;
+        margin-right: 1.2vw;
+        @include breakpoint($large-screens) {
+          display: block;
+        }
+      }
+      .slider__range {
+        align-self: center;
+        -webkit-appearance: none;
+        overflow: visible;
+        width: 100%;
+        height: 1em;
+        border-radius: 5px;
+        outline: none;
+        background-color: #DBDBDB;
+        &::-webkit-slider-thumb {
+          -webkit-appearance: none;
+          background: radial-gradient(circle farthest-corner at center center, $range-handle-color-center 20%, white 22%);
+          border-radius: 50px;
+          box-shadow: 0px 0px 48px rgba(0, 0, 0, 0.3);
+          @include fluid-size('width', $range-handle-size, $range-handle-size-big);
+          @include fluid-size('height', $range-handle-size, $range-handle-size-big);
+        }
+      }
+      .slider__range::before {
+        -webkit-appearance: none;
+        overflow: hidden;
+        color: #13bba4;
+      }
+    }
+    @include breakpoint($large-screens) {
+      order: -1;
```

Comment:
I have mixed feelings about this. I'm feeling as if someone used a !important in their CSS.